### PR TITLE
adding namespace and docs

### PR DESCRIPTION
**Pull request by puja108**

### CODE REVIEW

**Comment by puja108** [comment]
RFR @marians @webwurst

**Comment by marians** [comment]
Is the idea to add this to the docs as a guide?

I would remove `THUX` from the title and headline, since it's no longer self explanatory as an abbreviation. Maybe add a sentence somewhere early to explain that we call this example application "Twitter Hot URIs Example - short THUX".

**Comment by marians** [comment]
The docs content still explains the `resolver-scaler`, which isn't there any more I think.

**Comment by marians** [comment]
Also `rebrow` is still mentioned in the text.

**Comment by marians** [comment]
The component names in text and graphical overview differ in some parts.

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+printf "exampletokenxyz" | base64
+```
+
+## Deploying the Application {#deploy}
+
+__Note:__ To get custom metrics you need to have our [Monitoring recipe](/guides/kubernetes-prometheus/) deployed.
+
+Then, you can deploy the app by running:
+
+`kubectl apply --filename https://raw.githubusercontent.com/giantswarm/twitter-hot-urls-example/master/manifests-all.yaml`
```
Comment (marians) [inline docs/index.md]: I'd prefer a code block here instead of inline

**Comment by puja108** [comment]
Yes, I'd like to have this in docs as an example, will work on your fixes

**Comment by puja108** [comment]
Fixed according to @marians' comments. For `rebrow` I added that by default it is not deployed, I didn't remove it cause it's also in the overview.png

**Comment by marians** [comment]
BTW, I am available to change the overview image :)

**Comment by webwurst** [comment]
please remove `DEVNOTES.md` :)

**Comment by webwurst** [comment]
see there for current version to create `manifests-all.yaml`:
https://github.com/giantswarm/kubernetes-prometheus/blob/e1830e749452d769cc804976033886e7ed89fbc3/development.md

at the end there is no `---` anymore. still not sure if i need to add an additional `\n`.

**Comment by puja108** [comment]
done

**Comment by puja108** [comment]
aded some last minute fixes ;)

**Comment by puja108** [comment]
btw, I saw that rebrow is still deployed

**Review by marians — APPROVED** [review]
LGTM

**Pull request merged**